lady_luck.js

```diff
@@ -48,37 +38,29 @@ function d10RefillCheck() {
 	}
 }
 
-function nwodRoll(pool, options) {
+function nwodRoll(pool, options='') {
 	var again = 10;
 	var rote = false;
 	var botching = false;
-	var l = options ? options.length : 0;
-	for (i = 0; i < l; i++) {
-		if (options[i] == "8") {
-			again = 8;
-			continue;
-		}
 
-		if (options[i] == "9") {
-			again = 9;
-			continue;
-		}
-
-		if (options[i] == "r") {
-			rote = true;
-			continue;
-		}
-
-		if (options[i] == "b") {
-			botching = true;
-			continue;
-		}
-
-		if (options[i] == "n") {
-			again = 11;
-			continue;
+	for (const char of options)
+		switch (char) {
+			case '8':
+				again = 8;
+				break;
+			case '9':
+				again = 9;
+				break;
+			case 'r':
+				rote = true
+				break;
+			case 'b':
+				botching = true;
+				break;
+			case 'n':
+				again = 11;
+				break;
```

Comment:
No curly braces. Relying on indentation for code flow is _no bueno_.

Reply:
Switch `case` statements don't use brackets. Indentation is purely for readability

Reply:
`for` loops do, which is where the grip comes from. Look at line 46.